test/snippet/range/view/kmer_hash.cpp

```diff
@@ -1,14 +1,17 @@
-//! [usage]
 #include <seqan3/alphabet/nucleotide/dna4.hpp>
 #include <seqan3/core/debug_stream.hpp>
 #include <seqan3/range/view/kmer_hash.hpp>
 
-using namespace seqan3;
+using seqan3::operator""_dna4;
 
 int main()
 {
-    std::vector<dna4> text{"ACGTAGC"_dna4};
-    std::vector<size_t> hashes = text | view::kmer_hash(3);
-    debug_stream << hashes << '\n'; // [6,27,44,50,9]
+    std::vector<seqan3::dna4> text{"ACGTAGC"_dna4};
+    
+    std::vector<size_t> hashes = text | seqan3::view::kmer_hash(seqan3::shape(3));
```

Comment:
```suggestion
    std::vector<size_t> hashes = text | seqan3::view::kmer_hash(seqan3::shape(3));
```
Hm, it's really not good to mix a constructor that takes length and a constructor that takes a pack of the same type. Can you make a strong_type of the length parameter (maybe called it `ungapped`?).

Reply:
Now that I have a strong_type for this, I could add the literal constructor for `0b011010101` and stuff like that...

Reply:
Yes, I would make two strong types
  * `ungapped`
  * `pattern` or `bin_literal` (bin_literal might be usable in other contexts as well)

And then I would remove the parameter pack constructor, but that's up to you.

Reply:
In fact, you could even add a user defined numeric literal so one can do the following `0b11001101_shape` and it will return a shape. Pretty cool. But has room for confusion, because `5_shape` would also work and resolve to `0b101`...